specification/keyvault/Security.KeyVault.EKM/client.tsp

```diff
@@ -0,0 +1,88 @@
+import "./main.tsp";
+import "@azure-tools/typespec-client-generator-core";
+
+using Azure.ClientGenerator.Core;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+using TypeSpec.Versioning;
+
+@versioned(KeyVault.Versions)
+namespace Customizations;
+
+using KeyVault;
+
+/**
+ * The External Key Manager (EKM) Get operation returns EKM connection. This operation requires ekm/read permission.
+ */
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Foundations.Operation is necessary for Key Vault"
```

Comment:
For older endpoints, yes. Why can't you use them for new endpoints? While it's important to maintain consistency within a service, you weren't far off from what standard operation bases provide in terms of endpoint construction. Did you even try?